src/app.ts

```diff
@@ -6,17 +6,17 @@ import logger from 'koa-logger';
 import { DataFactory } from 'rdf-js';
 import Articles from './articles';
 import apiDocumentationLink from './middleware/api-documentation-link';
+import setDataFactory, { DataFactoryContext } from './middleware/data-factory';
 import emptyResponse from './middleware/empty-response';
 import errorHandler from './middleware/error-handler';
 import jsonld from './middleware/jsonld';
 import routing from './middleware/routing';
 
 export type AppState = DefaultState;
 
-export type AppContext = RouterContext<AppState, {
+export type AppContext = RouterContext<AppState, DataFactoryContext<{
```

Comment:
where's the type for `RouterContext`? DefinitelyTyped or somewhere else?

Reply:
Yep https://github.com/libero/article-store/blob/dd041f613487e7b39f72ba3c3a4f3fcc8e89e3da/src/app.ts#L2

Reply:
ah, I didn't see it in https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types because the list was too long